### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by StasOkhrym**

### CODE REVIEW

**Review by mltdll — APPROVED** [review]
Seems good to me.

File: app/test_main.py [hunk 1]
```diff
@@ -1,3 +1,55 @@
...
+                id="should return correct age with random numbers"
+            ),
+            pytest.param(
+                45, 18, [7, 1],
+                id="should return correct age with random numbers"
+            )
+        ]
+    )
+    def test_get_correct_age(
+            self,
```
Comment (Nattalli) [inline app/test_main.py:50]: I think that `self` is unneeded here

File: app/test_main.py [hunk 2]
```diff
@@ -1,3 +1,55 @@
...
+                -1, -1, [0, 0],
+                id="should return 0 if animal age is negative"
+            ),
+            pytest.param(
+                100, 100, [21, 17],
+                id="should work correct with big numbers"
+            ),
+            pytest.param(
+                24, 39, [2, 5],
+                id="should return correct age with random numbers"
```
Comment (Nattalli) [inline app/test_main.py:37]: can you change the id? It should be unique for each case

**Review by Nattalli — CHANGES_REQUESTED** [review]

**Review by Nattalli — APPROVED** [review]
Well done!

**Pull request closed without merging**